api/controllers/ApplicationController.js

```diff
@@ -436,31 +440,36 @@ var applicationController = {
      * @param req {Array} - request object
      * @return send to rabbitmq response
      */
-    exportAppData: function(req, res) {
-
-        var appId = req.query.merchantReturnData;
+    exportAppData: function(req, application) {
+        // populate_exportedeApostilleAppdata
+        const appId = req.query.merchantReturnData;
+        const isEApp = application.serviceType === 4;
+        const storedProdToUse = isEApp
+            ? 'populate_exportedeApostilleAppdata'
+            : 'populate_exportedapplicationdata';
         //Call postgres stored procedure to insert and returns 1 if successful or 0 if no insert occurred
-       sequelize.query('SELECT * FROM populate_exportedapplicationdata(' + appId + ')')
-         .then(function (results) {
-           sails.log("Application export to exports table completed.");
-           Application.update({
-             submitted: 'queued'
-           }, {
-             where: {
-               application_id: appId
-             }
-           })
-             .then(function () {
-               sails.log.info('queued ' + appId);
-             })
-             .catch(Sequelize.ValidationError, function (error) {
-               sails.log.error(error);
-             });
-         })
-         .catch(Sequelize.ValidationError, function (error) {
-           sails.log(error);
-         });
-    }
+        sails.log.info(appId + ' - exporting app data');
+        sequelize.query(`SELECT * FROM ${storedProdToUse}(${appId})`)
+            .then(function (results) {
+            sails.log.info("Application export to exports table completed.");
+            Application.update({
+                submitted: 'queued'
+            }, {
+                where: {
+                application_id: appId
+                }
+            })
+                .then(function () {
+                sails.log.info('queued ' + appId);
+                })
+                .catch(Sequelize.ValidationError, function (error) {
+                sails.log.error(error);
+                });
+            })
+            .catch(function (error) {
+            sails.log.error(error);
```

Comment:
nitpicking, is the indentation right here? have you run prettier against it? (or was that another PR)

Reply:
I'm a bit hesitant to run prettier on this file since most of the code wasn't written by me and wasn't originally formatted correctly so this PR would have been full of formatting changes making it difficult to see what has actually been changed.

But yeah it's a good spot, I hadn't noticed it before 👍